dist.ini

```diff
@@ -36,10 +36,7 @@ tag_format = %v
 push_to = origin master
 
 [AutoPrereqs]
-skip = ^DDG::*
-
-[Run::BeforeBuild]
-run = cpanm DDG --skip-installed --mirror http://www.cpan.org/ --mirror http://duckpan.org
+skip = ^DDG::(Spice|Goodie|Fathead|Longtail|Publisher)
```

Comment:
DuckPAN doesn't use/require any Role's so they don't need to be ignored as dependencies.

Technically Fathead and Longtail don't need to be here, but it doesn't really hurt.

Spice, Goodie and Publisher are all `use`d though, so it tries to install them